includes/options/class-amp-options-manager.php

```diff
@@ -259,4 +260,20 @@ public static function update_analytics_options( $data ) {
 		_deprecated_function( __METHOD__, '0.6', __CLASS__ . '::update_option' );
 		return self::update_option( 'analytics', wp_unslash( $data ) );
 	}
+
+	/**
+	 * Outputs an admin notice if persistent object cache is not present.
+	 *
+	 * @return void
+	 */
+	public static function persistent_object_caching_notice() {
+		if ( ! wp_using_ext_object_cache() && 'toplevel_page_' . self::OPTION_NAME === get_current_screen()->id ) {
```

Comment:
`'toplevel_page_' . self::OPTION_NAME` can be replaced with `get_plugin_page_hookname( self::OPTION_NAME, null )` to avoid hard coded prefix.

Reply:
I tried swapping that out and it worked but it broke the unit test. Feel free to amend the PR with the change.